partner_relations_in_tab/model/res_partner.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+from odoo.osv.orm import Model, transfer_modifiers_to_node
+from odoo.osv import expression
+from odoo.tools.translate import _
+
+
+class ResPartner(Model):
+    _inherit = 'res.partner'
+
+    def _get_relation_ids_select(self, field_name, arg):
```

Comment:
This function is not needed anymore. In the old version of partner_relations, all relations where shown in a tab. This override was to exclude the relations from the main tab, that were shown in their own tab.